cookiecutter/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenBB Platform Extensions Cookiecutter
+
+[Cookiecutter](https://cookiecutter.readthedocs.io/en/1.7.2/) is a command-line utility that creates projects from templates.
+
+This extension is a simple template for setting up new OpenBB Python Package extensions and projects.
+
+## Template Structure
+
+The Cookiecutter template prompts the user for information to use in the `pyproject.toml` file, and then generates a project based on that information.
+All fields are optional.
+
+- Your Name
+- Your Email
+- Project Name
+- Project Tag (some-distributable-package)
+- Package Name ("include" code folder name - "some_package")
+- Provider Name - name of the provider for the entry point - i.e, 'fmp'
+- Router Name - name of the router path - i.e. `obb.{some_package}`
+- OBBject Name - name of the OBBject accessor namespace.
+
+The template will generate all extension types as a single, installable Python project.
+You likely won't always use all in tandem, just delete the unwanted folders and entrypoints.
+
+## Usage
```

Comment:
Once it's published to pypi it allegedly can be used with the following command:

`uvx openbb-cookiecutter`